dart_conf_flutter/lib/src/schedule.dart

```diff
@@ -158,16 +157,14 @@ class Session implements Comparable<Session> {
 
   // nullable
   final Duration duration;
-  final Iterable<String> presenters;
+  final String presenters;
 
   final String imageUrl;
 
   bool get hasPresenters => presenters?.isNotEmpty ?? false;
 
   String get presentersDescription {
-    return presenters.length > 2
-        ? presenters.join(', ')
-        : presenters.join(' and ');
+    return presenters;
```

Comment:
you can also remove `presentersDescription` completely and change callers to use `presenters` directly

Reply:
Left presentersDescription, but changed it to return the empty string if no presenters specified.